seqr/views/apis/variant_search_api.py

```diff
@@ -129,28 +128,33 @@ def query_single_variant_handler(request, variant_id):
 
     variant = get_single_es_variant(families, variant_id, user=request.user)
 
-    response = _process_variants([variant], families, request.user)
+    response = _process_variants([variant], families, request)
     response.update(_get_projects_details([families.first().project], request.user))
 
     return create_json_response(response)
 
 
-def _process_variants(variants, families, user):
+def _process_variants(variants, families, request):
     if not variants:
         return {'searchedVariants': variants}
 
-    prefetch_related_objects(families, 'project')
-    genes = saved_variant_genes(variants)
-    projects = {family.project for family in families}
-    locus_lists_by_guid = add_locus_lists(projects, genes)
-    response_json, _ = _get_saved_variants(variants, families, include_discovery_tags=user_is_analyst(user))
-
-    response_json.update({
-        'searchedVariants': variants,
-        'genesById': genes,
-        'locusListsByGuid': locus_lists_by_guid,
-        'rnaSeqData': get_rna_seq_outliers(genes.keys(), sample__individual__family__in=families),
-    })
+    flat_variants = _flatten_variants(variants)
+    saved_variants, variants_by_id = _get_saved_variant_models(flat_variants, families)
+
+    response_json = get_variants_response(request, saved_variants, response_variants=flat_variants)
+    response_json['searchedVariants'] = variants
+
+    for saved_variant in response_json['savedVariantsByGuid'].values():
```

Comment:
this si copied from later in this file, from what was the `_get_saved_variants` helper function

Reply:
Compared to the old `_get_saved_variants` code, the new code doesn't update the `discoveryTags` of the searched variant. Is there any reason?

Reply:
So that code actually got moved inside `get_variants_response` so it is still being called : https://github.com/broadinstitute/seqr/pull/2461/files#diff-925641acb700e0420de6f62f724c1266e92711a3c7d9b7345a5899e5671423b0R171

We actually have an explicit test for this in `variant_earch_api_tests` that is still passing, which indicates that it is still working as expected